core/src/main/java/io/kestra/core/models/tasks/Task.java

```diff
@@ -114,4 +120,11 @@ public boolean isFlowable() {
     public boolean isSendToWorkerTask() {
         return !(this instanceof FlowableTask) || this instanceof WorkingDirectory;
     }
+
+    @Getter
+    @NoArgsConstructor
+    @AllArgsConstructor
+    public static class WorkerGroup  {
+        private String key;
```

Comment:
Should add some validation format since we will not support all the chars (based on the kafka limitation